internal/pkg/platform/platform_matcher.go

```diff
@@ -82,9 +83,25 @@ func getCPUVariantWindows(arch string) string {
 func getCPUVariantArm() string {
 	variant, err := getCPUInfo("Cpu architecture")
 	if err != nil {
+		logrus.Errorf("Couldn't get cpu architecture")
```

Comment:
If it makes sense to log this, I think the encountered error is definitely worth including as well.